ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,46 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+The OCF maintains a strict policy of not removing or modifying content
+hosted by user accounts unless legally compelled to do so, or if such
+content is in violation of existing OCF account policies.
+
+This means the OCF cannot, and will not, change or remove any text,
+pictures, information, or other content hosted by an OCF user. Only the
+user who owns the relevant OCF account can modify the data we host on
+their behalf. We do however reserve the right to deactivate inactive
+accounts at our discretion.
```

Comment:
Maybe Matt means "owner" as the person who uploaded it not the legal/copyright owner? Or that's what you mean? In any case I'll clarify the legal issues you mentioned in the other comments, which should help clarify this section as well. I agree though in general we don't want to be getting involved in anything that has legal ramifications including authenticating content ownership.